docs/root/api/client_features.rst

```diff
@@ -18,3 +18,6 @@ Currently Defined Client Features
   :ref:`overprovisioning_factor<envoy_api_field_ClusterLoadAssignment.Policy.overprovisioning_factor>`
   field. If graceful failover functionality is required, it must be supplied by the management
   server.
+- **envoy.lrs.supports_send_all_clusters**: This feature indicates that the client supports
```

Comment:
Do you plan on implementing this in Envoy? It's not a ton of work.

Reply:
If it's something I can do fairly quickly, I can take a stab at it.  Can you point me at the code that would need to be changed?

Reply:
It would mean changing the loop logic in https://github.com/envoyproxy/envoy/blob/master/source/common/upstream/load_stats_reporter.cc#L48, instead of looping over known clusters from LRS and looking them up in cluster manager, we would iterate over all the clusters in cluster manager in the wildcard case.

Reply:
Thanks for implementing; have you added the client feature capability announcement yet?

Reply:
Done.  Also added a bullet to the release notes about this.